node/libs/roles/src/validator/messages/msg.rs

```diff
@@ -14,6 +17,8 @@ pub enum Msg {
     SessionId(SessionId),
     /// validator discovery
     NetAddress(NetAddress),
+    /// l1 batch signature
+    L1BatchSignature(L1BatchSignatureMsg),
```

Comment:
Something went wrong here. It is the Msg what is being signed, so you want Msg::L1Batch variant not L1BatchSignature, otherwise validator will be signing its own signature which doesn't make sense in this case.